src/helpers.cpp

```diff
@@ -115,7 +115,10 @@ ada_really_inline bool shorten_path(std::string& path,
 
   // Remove path’s last item, if any.
   if (!path.empty()) {
-    path.erase(path.rfind('/'));
+    size_t last_delimiter = path.rfind('/');
+    if (last_delimiter != std::string::npos) {
+      path.erase(last_delimiter);
```

Comment:
I believe return true should be inside this if statement.

Reply:
@anonrig Yep.